ckan/config/middleware/flask_app.py

```diff
@@ -557,7 +615,7 @@ def filter(self, log_record: Any) -> bool:
 
 
 def _setup_webassets(app: CKANApp):
-    app.use_x_sendfile = config.get('ckan.webassets.use_x_sendfile')
+    setattr(app, "use_x_sendfile", config.get('ckan.webassets.use_x_sendfile'))
```

Comment:
why setattr? If this is only to avoid a pyright error I'd say leave the original because it's clearer

Reply:
Yes, that only added to suppress typechecker complaints, reverting it back